static/ckeditor/ckeditor/plugins/ajax/plugin.js

```
@@ -0,0 +1,241 @@
+		}
+
+		function getResponse( xhr, type ) {
+			if ( !checkStatus( xhr ) ) {
+				return null;
+			}
+
+			switch ( type ) {
+				case 'text':
+					return xhr.responseText;
+				case 'xml':
+					var xml = xhr.responseXML;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)